xbmc/cores/VideoPlayer/Interface/StreamInfo.h

```diff
@@ -30,6 +30,14 @@ enum StreamFlags
   FLAG_STILL_IMAGES = 0x100000
 };
 
+enum class StreamHdrType
```

Comment:
This is not a mandatory change but if you can go a step further in the documentation and guarantee that the infolabels are always in sync with the implementation you could document the enum "string values" here doing something similar to this:

https://github.com/xbmc/xbmc/blob/a80d1293b0579bbed7dd7d20304ba34a20519f78/xbmc/XBDateTime.h#L51-L74

Then in the infolabel documentation you can simply write:

``///   \table_row3{   <b>`ListItem.HdrType`</b>,
///                  \anchor ListItem_HdrType
///                  _string_,
///     @return String containing the name of the detected HDR type or empty if not HDR. See \ref StreamHdrType for ///     the list of possible values.
///     <p><hr>
///     @skinning_v20 **[New Infolabel]** \link ListItem_HdrType `ListItem.HdrType`\endlink
///   }`
```